server/controllers/inventory/index.js

```diff
@@ -88,6 +93,98 @@ function inventoryLog(req, res, next) {
   }).catch(next);
 }
 
+// get inventory log as excel
+// GET /inventory/download/log/:uuid?rendere=xlsx?lang=fr
+
+async function logDownLoad(req, res, next) {
```

Comment:
It would be nice to have an integration test for this functionality, to at least show it still works. Could you write one?